src/client/pages/OfferNew/utils.ts

```diff
@@ -93,33 +104,28 @@ const getAddressFromBundledQuotes = (
   return null
 }
 
-export const quoteDetailsHasAddress = (
+const isHomeQuote = (
   quoteDetails: QuoteDetails,
 ): quoteDetails is
   | SwedishApartmentQuoteDetails
   | SwedishHouseQuoteDetails
   | NorwegianHomeContentsDetails
   | DanishHomeContentsDetails =>
-  [
-    'SwedishApartmentQuoteDetails',
-    'SwedishHouseQuoteDetails',
-    'NorwegianHomeContentsDetails',
-    'DanishHomeContentsDetails',
-  ].includes(quoteDetails.__typename as string)
+  isNorwegianHomeContents(quoteDetails) ||
+  isDanishHomeContents(quoteDetails) ||
+  isSwedishHouse(quoteDetails) ||
+  isSwedishApartment(quoteDetails)
+
+export const quoteDetailsHasAddress = isHomeQuote
 
 export const getMainQuote = (offerData: OfferData) => {
-  const mainQuoteInBundle = offerData.quotes.filter((quote) => {
-    const isHomeContentsQuote =
-      isNorwegianHomeContents(quote.quoteDetails) ||
-      isDanishHomeContents(quote.quoteDetails)
-    return isHomeContentsQuote
-  })
+  const mainQuoteInBundle = offerData.quotes.find(({ quoteDetails }) =>
+    isHomeQuote(quoteDetails),
+  )
 
-  const mainQuote = isBundle(offerData)
-    ? mainQuoteInBundle[0]
-    : offerData.quotes[0]
+  const firstQuote = offerData.quotes[0]
 
-  return mainQuote
+  return (isBundle(offerData) && mainQuoteInBundle) || firstQuote
```

Comment:
Actually it might be bad to do it this way that if we have a bundle but for some reason we can't find the Home Quote we just return the first quote, since there should never ever be a case where we have a bundle without a home quote. So I'm leaning towards throwing an error or something instead if that's the case. WDYT?

Something like: 
```suggestion
const mainQuote = isBundle(offerData) ? mainQuoteInBundle : firstQuote

if (!mainQuote) {
  throw Error('There must be a main quote')
}
return mainQuote
```

or

```suggestion
if (isBundle(offerData)) {
    const mainQuoteInBundle = offerData.quotes.find(({ quoteDetails }) =>
    isHomeQuote(quoteDetails),
  )
  if (!mainQuoteInBundle) {
    throw Error('A bundle must include a home quote')
  }
  return mainQuoteInBundle
}
return offerData.quotes[0]
```

Reply:
I see your point and I agree that we should check for this case. I just see that this is not generic / scalable enough for the following reasons:
* I think that the offer page should be generic and be able display whatever combination of quotes we throw at it as long as they are valid. We are bound at some point in some market to support buying an insurance (or a bundle) that does not include home (Travel + Accident for example). 
* I think the responsibility to enforce this business rule should be (and is already) in other places such as the backend and UI for bundle selection.
Would be happy to discuss this more in person ☺️

Reply:
Yeah let't talk about it in person 👍

Reply:
@gustaveen We'd like to know your view on this so let's talk about it all three of us.

Reply:
Good point, it would be nice if backend could have those limitations and we will just output the results. I'm not sure that's the case right now though. Happy to talk more about it 🙌